iroha/src/asset.rs

```diff
@@ -207,7 +222,17 @@ pub mod isi {
     }
 
     impl Mint<Asset, u128> {
-        fn execute(&self, world_state_view: &mut WorldStateView) -> Result<(), String> {
+        fn execute(
+            &self,
+            authority: <Account as Identifiable>::Id,
+            world_state_view: &mut WorldStateView,
+        ) -> Result<(), String> {
+            PermissionInstruction::CanMintAsset(
```

Comment:
Does this consider the ability to mint a specific asset, rather than all assets?

Reply:
It consider the ability to Mint `Asset` of the specific `AssetDefinition`:
```rust
230             PermissionInstruction::CanMintAsset(
231                 authority,
232                 self.destination_id.definition_id.clone(),
233                 None,
234             )
```